src/Hooks/CheckoutScriptHooks.php

```diff
@@ -199,7 +199,17 @@ private function shouldShowDeliveryOptions(): bool
             /**  @var WC_Product $product */
             $product = $cartItem['data'];
 
-            if (! $product->is_virtual() && ! $product->is_on_backorder($cartItem['quantity'])) {
+            $productIsNotVirtual = ! $product->is_virtual();
```

Comment:
Voorstel tot verbetering, ga terug naar voor je [commit](https://github.com/myparcelnl/woocommerce/pull/1136/commits/ae1c35ac3ba291a392cb810bbbcf196d77e705f7) 'make if statement readable', want dat vond ik beter leesbaar dan dit... bovendien kun je je `$deliveryOptionsEnabledWhenNotInStock` graag buiten de loop door de producten zetten, dan hoeft ie de setting maar 1 keer op te halen en is ook logischer te lezen want ga je niet twijfelen of dat misschien een product setting is ofzo.